addons/website_slides/tests/test_slide_channel.py

```diff
@@ -121,6 +121,19 @@ def test_channel_partner_next_slide(self):
             # we have gone through all the content, next slide should be False
             self.assertFalse(membership.next_slide_id)
 
+    @users('user_officer')
+    def test_copy_slide_channel_enroll(self):
+        """Test copying a slide.channel and ensure the enroll field is copied."""
```

Comment:
This is a bit misleading as we won't be copying the `enroll` field in all cases.
``` suggestion
    def test_channel_enroll_policy_copy_with_visibility_member(self):
        """Make sure that `enroll` is preserved when copying a `visibility="members"` course ."""
``` 
(mute if included in other test)